src/components/AddTodo/AddTodo.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { TextField, Button, Box } from '@mui/material';
+import { makeStyles, createStyles } from '@mui/styles';
+import { useDispatch } from 'react-redux';
+import { addTodo } from '../../redux/todosSlice';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    addTodo: {
+      margin: '0 auto',
+    },
+  })
+);
+
+const AddTodo: React.FC = () => {
+  const [todoText, setTodoText] = useState('');
+  const dispatch = useDispatch();
+  const classes = useStyles();
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (todoText.trim()) {
+      dispatch(addTodo(todoText));
+      setTodoText('');
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTodoText(event.target.value);
+  };
+
+
+
+
```

Comment:
To make your code clean you should pay attention to how it's formatted. For example, this long space breaks the whole code reading flow. It seems like a minor problem, but actually annoys the hell out of reviewers

Reply:
The 'no-multiple-empty-lines' esLint rule has been added. And also I added Husky pre-commit to run linter and prettier.